src/species_advance/species_advance.h

```diff
@@ -611,8 +611,11 @@ move_p_kokkos(
       //(&(pm->dispx))[axis] = -(&(pm->dispx))[axis];
       //k_local_particle_movers(0, particle_mover_var::dispx + axis) = -k_local_particle_movers(0, particle_mover_var::dispx + axis);
       // TODO: replace this, it's horrible
```

Comment:
Now that this is done, we should get rid of this TODO.

Reply:
I've removed the TODO and made the same change in the serial version of move_p for consistency.